src/to_mathlib/analysis/inner_product_space/tensor_product.lean

```diff
@@ -10,6 +10,65 @@ open_locale real_inner_product_space tensor_product big_operators
 
 namespace inner_product_space
 
+lemma tensor_sum {α β H I : Type*} {α : Type*}
```

Comment:
Mathlib already has this lemma, see https://leanprover-community.github.io/mathlib_docs/linear_algebra/tensor_product.html#tensor_product.sum_tmul